CONTRIBUTING.md

```diff
@@ -92,6 +92,15 @@ Maintainers are:
  * @stu-gott
  * @vladikr
 
+### Becoming a memeber
+
+Contributors that frequently contribute to the project may ask to join the
+kubevirt organization.
```

Comment:
Can you add a word on the benefits of being a member? Can you get CI running automatically? Any other permission granted for members?

Reply:
It may be in our best interest to leave vague.
This may change with time and not committing to specifics is not always bad.

Reply:
ok, don't update the doc, but would you remind me: why would one currently want to be a member.

Reply:
I just wanted to be added as a reviewer or to be mentioned in comments.